### ISSUE TITLE
Fix request: Combination of annotations Lombok/Spring @Value false positive 

### ISSUE THREAD

**Issue by MarkvanOsch** [jborgers/PMD-jPinpoint-rules#123]
I see this jpinpoint rule for not combining Lombok annotations:

"Don't combine [@Data with @Value] and [@Data or @Value] with any of [@ToString, @EqualsHashCode, @Getter, @Setter, @RequiredArgsConstructor] (Lombok)"

But the @Value we are using here is the Spring annotation, not the Lombok annotation. So I consider this as a false positive.

Code snippet:
```
    @Getter
    @Value("${fpx.security.enabled:true}")
    private boolean securityEnabled;
```

**Comment by jborgers** [comment]
The checks should only be on class / inner class level, not fields level.
